packages/ui-overlays/src/Mask/index.tsx

```diff
@@ -72,7 +74,10 @@ class Mask extends Component<MaskProps> {
 
   componentWillUnmount() {
     if (this.props.fullscreen) {
-      noScroll.off()
+      MaskCounter.decrementCounter()
+      if (MaskCounter.getCounter() === 0) {
```

Comment:
Just to be extra careful, I'd check for counter being less than 1 (in case of rapid closure oof more than one modals)

Reply:
Good catch, thanks!